### DISCUSSION TITLE
jb install failure with 404

### DISCUSSION THREAD

**Discussion by TimothySutton81** [prometheus-operator/kube-prometheus#2416] [General]
I am getting a 404 when trying to do follow the customization. Has anyone else seen this or have an idea how to debug this issue? 

jb install github.com/prometheus-operator/kube-prometheus/jsonnet/kube-prometheus@main
fatal: unable to access 'https://github.com/prometheus-operator/kube-prometheus.git/': Problem (3) in the Chunked-Encoded data
GET https://github.com/prometheus-operator/kube-prometheus/archive/.tar.gz 404
archive install failed: unexpected status code 404
retrying with git...
Initialized empty Git repository in /home/fensAdmin/my-kube-prometheus/vendor/.tmp/c7908ed60dd6df49df7454e0d0064fc8641348449/.git/
remote: Enumerating objects: 1825, done.
remote: Counting objects: 100% (1825/1825), done.
remote: Compressing objects: 100% (1212/1212), done.
remote: Total 1825 (delta 1276), reused 916 (delta 556), pack-reused 0
Receiving objects: 100% (1825/1825), 1.11 MiB | 0 bytes/s, done.
Resolving deltas: 100% (1276/1276), done.
From https://github.com/prometheus-operator/kube-prometheus
 * branch            main       -> FETCH_HEAD
 * [new tag]         v0.1.0     -> v0.1.0
 * [new tag]         v0.10.0    -> v0.10.0
 * [new tag]         v0.11.0    -> v0.11.0
 * [new tag]         v0.12.0    -> v0.12.0
 * [new tag]         v0.13.0    -> v0.13.0
 * [new tag]         v0.2.0     -> v0.2.0
 * [new tag]         v0.3.0     -> v0.3.0
 * [new tag]         v0.4.0     -> v0.4.0
 * [new tag]         v0.5.0     -> v0.5.0
 * [new tag]         v0.6.0     -> v0.6.0
 * [new tag]         v0.7.0     -> v0.7.0
 * [new tag]         v0.8.0     -> v0.8.0
 * [new tag]         v0.9.0     -> v0.9.0
error: pathspec 'main' did not match any file(s) known to git.
jb: error: failed to install packages: downloading: exit status 1

**Comment by ayyappa460** [comment]
i have the same issue. do we have any solution for it.
